SingularityUI/app/actions/api/users.es6

```diff
@@ -0,0 +1,36 @@
+import { buildApiAction, buildJsonApiAction } from './base';
+
+export const FetchUserSettings = buildApiAction(
+  'FETCH_USER_SETTINGS',
+  userId => ({
+    url: `/users/settings?userId=${userId}`,
+    catchStatusCodes: [404] // Remove this once the API doesn't 404 when user has nothing set
+  })
+);
+
+export const UpdateUserSettings = buildJsonApiAction(
+  'UPDATE_USER_SETTINGS',
+  'POST',
+  (userId, settings) => ({
+    url: `/users/settings?userId=${userId}`,
```

Comment:
@MattCCS and @Calvinp , talked with @tpetr and we will remove the use of the query param from here. Matt, we will rely just on the injected `Optional<SingularityUser>` to determine the userId. (i.e. this will work only when using a SingularityAuthenticator setup besides the disabled one). Calvin, in the case where we aren't able to get the user settings from the api, we can fall back to using localStorage.